x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -173,6 +179,23 @@ export const EndpointList = () => {
     <BackToExternalAppButton {...backLinkOptions} data-test-subj="endpointListBackLink" />
   );
 
+  const missingFleetAccessInfo = useMemo(() => {
+    return (
+      <EuiText size="s" color="subdued" data-test-subj="noFleetAccess">
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.onboarding.enableFleetAccess"
+          defaultMessage="Deploying Agents for the first time requires Fleet access. For more information, "
+        />
+        <EuiLink external href={`${services?.docLinks.links.securitySolution.privileges}`}>
```

Comment:
I see `?` in the value here, which mean that there is the possibility for `services?.docLinks.links.securitySolution.privileges` to be `undefined`. Maybe wrap the `<EuiLink...>` in a conditional to protect against it? LIke:

```typescript
{ services?.docLinks.links.securitySolution.privileges && (
    <EuiLink ..... >
)}
```